src/Silk/Commands/Moderation/KickCommand.cs

```diff
@@ -42,7 +42,7 @@ IDiscordRestChannelAPI channels
     public async Task<IResult> Kick([NonSelfActionable] IUser user, [Greedy] string reason = "Not given.")
     {
         var infractionResult = await _infractions.KickAsync(_context.GuildID.Value, user.ID, _context.User.ID, reason);
-        var notified         = infractionResult.Entity.UserNotified ? "(User notified via DM)" : "(Failed to DM)";
+        var notified         = infractionResult.Entity.Notified ? "(User notified via DM)" : "(Failed to DM)";
```

Comment:
*May* be worth it to extract `success` and `failure` messages to a string constant? (could be used here and other places with the same success and failure text: `InfractionResultReplyText` or something 🚀

Reply:
Fair enough. I'll do this on the main branch as it's slightly outside the scope of this PR